src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -77,7 +77,7 @@
         "phpunit/phpunit": "<5.4.3",
         "symfony/asset": "<5.3",
         "symfony/console": "<5.2.5",
-        "symfony/dotenv": "<5.1",
```

Comment:
This bump should not be necessary anymore because of the `class_exists(DebugCommand::class)` safeguard.